README.md

```diff
@@ -1 +1,58 @@
-# git-remotes-with-github-readme
+#Git Remotes + Github
+
+##Objectives
+1. Describe Github and its relationship with git
+2. Create a remote repository on Github
+3. Use git push to connect your local repository of files to your remote repository 
+4. Use git remote
+5. Use git push
+6. Use git pull
+
+adapt from https://github.com/learn-co-curriculum/git-todo (2nd half) and from https://github.com/learn-co-curriculum/git-github-and-learn
+
+##Why this is useful
+Github does nothing special in the git universe. It's just another git repository in the cloud. IF you don't want to work with anyone else, you don't need remotes. However, this is rarely the case, and we want to work with others! So, we have to talk to remotes. 
+
+##Creating a remote repository on Github
+
+1. From your Github profile, click the "Create New Repository" button.
+2. After you create the repo, click the "Copy to clipboard" symbol on the right-hand side of the screen to copy the url (we'll use this in the next section)
+
+
+##Connecting your remote repo to a local repo
+1. `git --init`
+2. `git add .` + `git commit -m "initialize git"`. Add and commit the new file created by the init command in the step 1.
+3. `git remote add origin your-remote-repository-URL`. This sets the remote, so you can push and pull code.
```

Comment:
gotta unpack that remote command.

What is origin? Origin is the name of the repo, but if could be anything (show them that). Origin is just a convention for the default.